src/main/java/wolox/training/models/Book.java

```diff
@@ -0,0 +1,115 @@
+package wolox.training.models;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+
+@Entity
+public class Book {
+    @Id
```

Comment:
Create a field called id of type Long and make it self-incrementing

Reply:
I added a new Id field